### DISCUSSION TITLE
How Do I change the color of "all-day" event

### DISCUSSION THREAD

**Discussion by jerrybroughton** [niccokunzmann/open-web-calendar#394] [Q&A]
What CSS controls the color of "all-day" event. It currently is always light blue.  Also I noticed in the examples that Multiday "All-Day" events dont show accross days in things like the Availability example..  It does work correctly in the default examples.

**Comment by niccokunzmann** [comment]
> What CSS controls the color of "all-day" event. 

This are the classes of an event at a certain time:

```
dhx_cal_event_clear dhx_cal_event_line_start dhx_cal_event_line_end event UID-3n7klfc5jttu38k2qlv3a6bu0m@google.com TRANSP-OPAQUE STATUS-CONFIRMED CALENDAR-INDEX-2
```

This are the classes of an event that spans several days:

```
dhx_cal_event_line dhx_cal_event_line_start dhx_cal_event_line_end event UID-73oe8dj190cikb2eja1355mra2@google.com TRANSP-TRANSPARENT STATUS-CONFIRMED CALENDAR-INDEX-2
```

These are the classes of an event that spans one day:

```
dhx_cal_event_line dhx_cal_event_line_start dhx_cal_event_line_end event UID-83dfaef1-a308-455d-ba52-1b3756e97933 TRANSP-OPAQUE CALENDAR-INDEX-3
```

So, I think the CSS class for day events is `dhx_cal_event_line` and the CSS class for an event at a certain time is `dhx_cal_event_clear`. Could you check if it works and report back?

**Comment by niccokunzmann** [comment]
> Also I noticed in the examples that Multiday "All-Day" events dont show accross days in things like the Availability example.. It does work correctly in the default examples.

Hm. Maybe, when I made this example, I did not consider that.

So, in order to use this example, one could make an event that repeats every day and remove some events. Do you have an ICS file for this?
Or the width can be removed from the example and just the height adjusted.
